spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/web/servlet/ServletManagementChildContextConfiguration.java

```diff
@@ -83,20 +83,22 @@ ServletManagementWebServerFactoryCustomizer servletManagementWebServerFactoryCus
 
 	@Bean
 	@ConditionalOnClass(name = "io.undertow.Undertow")
-	UndertowAccessLogCustomizer undertowManagementAccessLogCustomizer() {
-		return new UndertowAccessLogCustomizer();
+	UndertowAccessLogCustomizer undertowManagementAccessLogCustomizer(
```

Comment:
It works without for the moment. Is it Undertow specific ?

Reply:
No, I guess something else imports the configuration properties, so that this isn't a problem. But generally speaking, every configuration (or bean) which uses configuration properties should enable them with `@EnableConfigurationProperties` to be on the safe side.